### PR TITLE
changing labels on examples (where incorrect) and making validate.js …

### PR DESCRIPTION
**Pull request by danfernand**
…a peerDep

### CODE REVIEW

**Comment by evanlucas** [comment]
Why is validate.js being made a peerDependency?

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "static": "static ./examples",
     "test": "browserify test/all.js -t babelify | tape-run",
     "build": "babel src --out-dir lib",
-    "prepublish": "npm run clean && npm run build",
+    "prepublish": "npm test && npm run clean && npm run build",
```
Comment (dkoston) [inline package.json]: :+1:

**Comment by dkoston** [comment]
LGTM

**Comment by dkoston** [comment]
@evanlucas cause you'll be using validate.js for form validation if you're using this package. This isn't a plugin per say but you'll likely be using validate.js here and in other parts of your project. Since you'll be interacting with validate.js to write your validation rules, it's not a private dependency

**Comment by evanlucas** [comment]
But we are using it in `src/reducers.js`???

**Comment by evanlucas** [comment]
So, this is also technically a dependency of this package as well

**Comment by dkoston** [comment]
Yea, we call validate() but you need to roll your own rules like in: https://github.com/helpdotcom/redux-react-form/blob/master/examples/rules/registration-form.js

Was talking with @danfernand this morning and we should dependency inject the validator in an upcoming release.

**Comment by evanlucas** [comment]
That's fine. With the way that it currently is though, it should still be a regular dependency as we are importing and this would break with npm@>=3 as peerDependencies are not automatically installed.

**Comment by dkoston** [comment]
good point. @danfernand, the README should be updated to reflect this as well until such a time that the form isn't tied to a specific validation library

**Comment by danfernand** [comment]
modified readme to express validate.js as a peer dep

**Comment by dkoston** [comment]
thanks, LGTM

**Pull request merged**